app/src/main/res/values/strings.xml

```diff
@@ -52,13 +52,19 @@
     <string name="city">City:</string>
     <string name="address">Address:</string>
     <string name="tell_about_yourself">Tell us about yourself:</string>
+    <string name="found_neighbors">we believe that your neighbors can help you out</string>
     <string name="we_found_neighbors">we found neighbors that have the item you were looking for</string>
-    <string name="isResolvedTitle">Is Resolved?</string>
-    <string name="request_resolved_message">Glad it worked out!
-        Say thanks by sending out a badge</string>
+    <string name="isResolvedTitle">Problem solved?</string>
+    <string name="request_resolved_message">Say thanks by sending out a badge!</string>
     <string name="neighbor_offering_help_msg">I can help you out :)</string>
     <string name="my_requests">My requests:</string>
     <string name="askToAddNewItem">You are awsome for helping!</string>
     <string name="add_item_to_the_inventory_to_help_others">Add item to the inventory to help others :)</string>
     <string name="next_time">next time</string>
+    <string name="done">done!</string>
+    <string name="send">Send!</string>
+    <string name="request_resolved_title">You have such great neighbors!</string>
+    <string name="description_hint">ex. 5 steps ladder</string>
```

Comment:
isn't this duplicate for item_example?

Reply:
nope, I changed the name hint to ladder, and the description hint to  this.. 
It's because we don't handle multiple words in a name so good :)